Code/Tools/ProjectManager/Source/ExternalLinkDialog.cpp

```diff
@@ -85,7 +85,6 @@ namespace O3DE::ProjectManager
 
     void ExternalLinkDialog::SetSkipDialogSetting(bool state)
     {
-        auto settings = SettingsInterface::Get();
-        settings->Set(QString(settings->ExternalLinkWarningKey), state);
+        SettingsInterface::Get()->Set(QString(ISettings::ExternalLinkWarningKey), state);
```

Comment:
I'm kinda surprised you need to create the QString() here, would think the QString char * constructor would pick it up for you.